app/src/processing/app/Editor.java

```diff
@@ -1515,6 +1513,21 @@ public void actionPerformed(ActionEvent e) {
       menu.add(useSelectionForFindItem);
     }
 
+    menu.addMenuListener(new MenuListener() {
+      @Override
+      public void menuSelected(MenuEvent e) {
+        boolean enabled = textarea.getSelectedText() != null;
+        cutItem.setEnabled(enabled);
+        copyItem.setEnabled(enabled);
+      }
+
+      @Override
+      public void menuDeselected(MenuEvent e) {}
```

Comment:
Yes. Because I'm instancing an anonymous class from MenuListener, it's mandatory to implement every method. But I choose use only menuSelected for checking if there's selected text.